timemachine/fe/free_energy.py

```diff
@@ -452,7 +455,19 @@ def prepare_host_edge(self, ff: Forcefield, host_config: HostConfig, lamb: float
         host_bps, host_masses = openmm_deserializer.deserialize_system(host_config.omm_system, cutoff=1.2)
         hgt = topology.HostGuestTopology(host_bps, self.top, host_config.num_water_atoms, ff, host_config.omm_topology)
 
-        final_params, final_potentials = self._get_system_params_and_potentials(ff_params, hgt, lamb)
+        final_params = []
```

Comment:
This is a definitely a cludge, but briefly looking at trying to emulate the behavior of the SingleTopology class seemed like a lot more details to get right than to just unwrap the potentials. This way the final potentials are in the same structure as what is constructed using the `SingleTopology.combine_with_host`.